src/gobstuf/stuf/brp/base_response.py

```diff
@@ -73,12 +74,20 @@ class StufMappedResponse(StufResponse):
     # Class properties to pass to the mapped object filter
     filter_kwargs = []
 
+    response_type = None
+
     def __init__(self, msg: str, **kwargs):
         if 'expand' in kwargs:
             self.expand = kwargs['expand'].split(',') if kwargs['expand'] else []
             del kwargs['expand']
         else:
             self.expand = []
+
+        # If the response type is one of Related detail or list, we need to expand the specific relation
+        if isinstance(self.response_type, (RelatedDetailResponse, RelatedListResponse)):
+            expand_parameter = self.response_type.related_type
```

Comment:
Can you not give the response a property that we can test. It makes the code more maintainable and uses polymorphism to start an action.

Reply:
Changed the code to be more generic. Response filters can be added to a response template